### PR TITLE
(FEAT) Changed Files Capability

### PR DESCRIPTION
**Pull request by johnmccabe**
Making this visible as discussed offline, this is a capability I put together a short while ago which exposes files changed in a running container.

screen shot 2017-06-01 at 18 01 40

### CODE REVIEW

**Comment by rick** [comment]
This is looking good, code-wise (I think there's like one `TODO` in there, whatever). Information is super useful, imo. Here's a report against the local docker-composed version of CI.next:  https://reporter.app.lumogon.com/5SBYmP81tmAUNyylI-qZLyBYdq7X6xecQu8TNwTTG9s=

**Comment by johnmccabe** [comment]
@tylerpace I understand you had some bother building this PR, can you try against after this mergeup?

**Comment by tylerpace** [comment]
@johnmccabe I'm able to build now and create reports with the new capability, thanks for the merge.

File: dockeradapter/dockeradapter.go [hunk 1]
```diff
@@ -98,6 +101,11 @@ type CopyFrom interface {
 	CopyFromContainer(ctx context.Context, container, srcPath string, followSymlink bool) (io.ReadCloser, dockertypes.ContainerPathStat, error)
 }
 
+// Diff TODO
```
Comment (garethr) [inline dockeradapter/dockeradapter.go]: Looks like a left-over TODO? Or should this be expanded?

**Comment by garethr** [comment]
Looks like this is currently missing any tests to exercise the capability?

**Comment by garethr** [comment]
@johnmccabe if you could also prefix the commits with (TC-151) now we have a suitable identifier for this

**Comment by johnmccabe** [comment]
@garethr can you have a look please, should be good to :shipit:

**Pull request merged**